cfy_manager/main.py

```diff
@@ -238,7 +239,7 @@ def brokers_add(**kwargs):
     """
     setup_console_logger(verbose=kwargs['verbose'])
     config.load_config(kwargs.get('config_file'))
-    _validate_components_prepared('brokers_add')
+    _validate_components_prepared('brokers_add', kwargs.get('config_file'))
```

Comment:
I think we need to have a way of passing a list of services to this as well, at least one of which must be configured?
e.g. for this one it's not enough that all the configured components are prepared, if only the database component is configured?